index.d.ts

```diff
@@ -0,0 +1,56 @@
+declare module 'speaker' {
```

Comment:
You don't need this block. In fact, using it will declare a global module instead of describing the corresponding file (`index.js`)

Reply:
Really? All of my TS typings I always did that way since I saw it how it's done. I remove it.

Reply:
It's only needed when declaring types for another module, e.g. if you would publish a separate package with the types (`speaker-typings`) then you would have to have it ☺️